### ISSUE TITLE
Inconsistent styling of Chinantec tone marks

### ISSUE THREAD

**Issue by dscorbett** [googlefonts/noto-fonts#1730]
### Font
NotoSans-Regular.otf
NotoSansMono-Regular.otf
NotoSerif-Regular.otf

### Where the font came from, and when
Site: https://github.com/googlefonts/noto-fonts/blob/fcf8d5d5568fc96efcc4cbe6f9fbd402f62de98a/phaseIII_only/unhinted/otf/NotoSans/NotoSans-Regular.otf
Site: https://github.com/googlefonts/noto-fonts/blob/fcf8d5d5568fc96efcc4cbe6f9fbd402f62de98a/phaseIII_only/unhinted/otf/NotoSansMono/NotoSansMono-Regular.otf
Site: https://github.com/googlefonts/noto-fonts/blob/fcf8d5d5568fc96efcc4cbe6f9fbd402f62de98a/phaseIII_only/unhinted/otf/NotoSerif/NotoSerif-Regular.otf
Date: 2020-05-01

### Font version
Version 2.003

### Issue
In Chinantec (language system tag CCHN) the modifier letters U+02C8..U+02CB should be made consistent in length, vertical position, and stroke width with U+A717..U+A719. According to [L2/04-349R](https://www.unicode.org/L2/L2004/04349r-chinantec.pdf), they are all part of the same series of tone marks.

### Character data
ˈˉˊˋꜗꜘꜙ
U+02C8 MODIFIER LETTER VERTICAL LINE
U+02C9 MODIFIER LETTER MACRON
U+02CA MODIFIER LETTER ACUTE ACCENT
U+02CB MODIFIER LETTER GRAVE ACCENT
U+A717 MODIFIER LETTER DOT VERTICAL BAR
U+A718 MODIFIER LETTER DOT SLASH
U+A719 MODIFIER LETTER DOT HORIZONTAL BAR

### Screenshot
![ˈˉˊˋꜗꜘꜙ](https://user-images.githubusercontent.com/1124347/80842474-aae75580-8bcf-11ea-9583-18cc2db0e753.png)
![ˈˉˊˋꜗꜘꜙ](https://user-images.githubusercontent.com/1124347/80842475-aae75580-8bcf-11ea-9397-0ec80f488df6.png)
![ˈˉˊˋꜗꜘꜙ](https://user-images.githubusercontent.com/1124347/80842476-aae75580-8bcf-11ea-96ec-b6d6583e14b6.png)

**Comment by nizarsq** [comment]
I can see the vertical position of U+02C8 is lower and U+A717 is higher comparing with the rest, I also can see U+02CA and U+02CB length are not matching the rest, I'm not sure though if this the issue reported.
Now the question is, what is the right way to test those tones?  will a sequence of standalone tones (modifier) is sufficient to tell if there is an issue? or we need them together with Chinantec script? I copied some text that have tones from this [pdf](https://ebible.org/pdf/chzNTps/chzNTps_all.pdf) to generate test case to be able to see if they are inconsistent, but unfortunately I can't tell. 
@marekjez86, maybe language/script expert opinion will be helpful on this issue.
<img width="642" alt="Screen Shot 2020-06-27 at 12 39 23 AM" src="https://user-images.githubusercontent.com/22158882/85917409-b8573f80-b80e-11ea-968a-3f01afa8f015.png">
<img width="1290" alt="Screen Shot 2020-06-27 at 12 42 42 AM" src="https://user-images.githubusercontent.com/22158882/85917472-2e5ba680-b80f-11ea-9210-601bab069bb1.png">

**Comment by dscorbett** [comment]
You either need to set the language system tag to CCHN (as I did) or set the HTML `lang` tag to a Chinantec language, e.g. `lang="cco"`. Either way, the inconsistency will be evident.

**Comment by nizarsq** [comment]
> You either need to set the language system tag to CCHN (as I did) or set the HTML `lang` tag to a Chinantec language, e.g. `lang="cco"`. Either way, the inconsistency will be evident.

Screenshots I attached in my previous comment lang tag already have lang="cco. Could you test it again? Latin Greek Cyrillic fonts updated recently.

**Comment by moyogo** [comment]
There’s no CCHN language system tag in the sources or in the new TTF.

**Comment by dscorbett** [comment]
Exactly. @nizarsq, I’m not sure why you want me to retest this: the fonts cannot possibly support this properly without CCHN language system tags, and your screenshot clearly shows that the tone marks are still inconsistent.

**Comment by nizarsq** [comment]
> Exactly. @nizarsq, I’m not sure why you want me to retest this: the fonts cannot possibly support this properly without CCHN language system tags, and your screenshot clearly shows that the tone marks are still inconsistent.
code.

There are 2 reasons why I asked to retest:
1- to see how tone marks are displayed with actual text (Not stack of tone marks), this will give more realistic result
2- to test tone marks display using the updated font

In regard to language code, I confirmed that I used the language code cco.
